src/main/java/frc/robot/RobotContainer.java

```diff
@@ -16,6 +16,7 @@
 import frc.robot.subsystems.drive.CommandSwerveDrivetrain;
 
 public class RobotContainer {
+    private RobotContainer robotContainer;
```

Comment:
RobotContainer shouldn't be a private variable of itself - let's delete this line